### PR TITLE
 Add the support for chrome cast

### PR DESCRIPTION
**Pull request by 2Pacalypse-**
Currently, we're restricting playback on only one device at a time;
if chromecast is connected, then the radio will play on chromecast,
and if it's not, then the playback will happen locally.

I guess this might not be what some people want; for example,  if
they want to play radio both on the chromecast and locally at the
same time for some reason, so we can change that at a later date
after getting some feedback.

There are also some additional improvements that could be made, like
controling the volume on the chromecast player through the website,
but leaving those for some other time.

### CODE REVIEW

File: caster.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    if (this.remotePlayer.isConnected) return this.emit('cast:start');
+    return this.emit('cast:stop');
+  }
+
+  togglePause(isPaused) {
+    this.isPlaying = !isPaused.value;
+  }
+
+  play() {
+    const mediaInfo = new MediaInfo(this.mediaSource.location, 'audio/ogg');
```
Comment (vladimyr) [inline caster.js]: Hardcoding `audio/ogg`? 🤔

Comment (2Pacalypse-) [reply caster.js]: Yeah, missed that one during refactor.

Though not sure what's the best way to handle this; ideally, MIME type would be inferred from the codec data, but that might be a bit involved to implement.

There are quite a few `audio` MIME types: https://www.iana.org/assignments/media-types/media-types.xhtml#audio
So we would need a way to do some kind of mapping between codec data -> MIME type, and default to `audio/ogg` or something? Though in the end, how important is it really for the MIME type to be 100% correct? I think browsers will handle them relatively the same, as long as it's an `audio/*` that the browser supports: https://developer.mozilla.org/en-US/docs/Web/HTML/Supported_media_formats#Browser_compatibility.

What are your thoughts?

Comment (vladimyr) [reply caster.js]: Agree with everything you said. Tested with wildcard mimetype `audio/*` and it worked flawlessly 🎉

**Comment by vladimyr** [comment]
> There are also some additional improvements that could be made, like
> controling the volume on the chromecast player through the website,
> but leaving those for some other time.

Actually if you cast from your phone Android will give you ability to control chromecast volume through your phone volume controls so it is usable after all 🙃

**Comment by vladimyr** [comment]
@2Pacalypse- Deployed, we are live 👍

**Comment by 2Pacalypse-** [comment]
:tada:

**Pull request merged**